### PR TITLE
Initial Spanish language support

### PR DESCRIPTION
**Pull request by theresaanna**
This PR will add basic Spanish language support to the site.

### CODE REVIEW

**Comment by theresaanna** [comment]
Currently, one can choose to serve either the English or Spanish versions of the site, but not both. Going to continue to work to find a solution for this. 

@rogeruiz When you are back working on vote, I could use your help here.

**Comment by rogeruiz** [comment]
Sure thing @theresaanna. I'll make some comments on the lines before Monday so we have something to pair on. 👍

File: config.toml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 baseurl = "https://vote.usa.gov/"
```
Comment (rogeruiz) [inline config.toml]: @theresaanna Are we no longer using this `config.toml` for the build?

File: config/gulp/build.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ gulp.task('website', [ 'build', 'watch' ], function (done) {
     buildDrafts = '';
   }
 
-  var hugo = spawn('hugo', [ 'server', buildDrafts ]);
+  var hugo = spawn('hugo', [ 'server', buildDrafts, '--config=config_es.toml' ]);
```
Comment (rogeruiz) [inline config/gulp/build.js]: I love this idea of passing in the `--config=` param. Could we try something similar to the `SITE_BASEURL` that can get passed in via the environment before running `npm run build`, etc.

File: layouts/index.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
       <!-- Register BEGIN -->
       <div class="usa-grid page-inner">
         <header>
-          <h1 class="usa-display">Register to vote</h1>
+          <h1 class="usa-display">{{ ( index $.Site.Data.translations $.Site.Params.lang ).homepageheader }}</h1>
```
Comment (rogeruiz) [inline layouts/index.html]: 🎉

**Comment by rogeruiz** [comment]
I left some comments above. My idea for how we'd build the site with languages is in line with the work here.

1. A separate `config-lang.toml`, maybe with different directories for content.
1. Each builds into a `tmp/lang-*` directory sequentially.
1. A final build step merges all these `tmp/lang-*` into the `public/` directory.
1. `cf push` the `public/` directory up.

**Comment by rogeruiz** [comment]
__note__

The language switcher functionality will not work during local development of the site and it can only be tested in a deployed environment such as `staging` and `production`.

**Comment by rogeruiz** [comment]
D'oh! there are conflicts. Let me try to resolve them @theresaanna.

**Pull request merged**